src_c/event.c

```diff
@@ -157,6 +157,14 @@ _pg_remove_pending_PGS_VIDEOEXPOSE(void * userdata, SDL_Event *event)
     return 1;
 }
 
+static int
```

Comment:
Small note: C already has a function, `int abs(int n)`, defined in `<stdlib.h>`, that does exactly what this function is doing (see: https://en.cppreference.com/w/c/numeric/math/abs).

Reply:
Hmmm ok, I didn’t know it was there. Anyways, we can save an include by defining the function ourselves right ;)

Reply:
There is also SDL_abs.